### DISCUSSION TITLE
[HW1] Are we not allowed to use a batch size of 1?

### DISCUSSION THREAD

**Discussion by iJinjin** [seominjoon/kaist-ai605#6] [Q&A] [answered]
Hello professor,

 In discussion #4, you said that we are freely allowed to modify the base architecture to make things work. 
Are we also allowed to just retain the baseline module as is and use a batch size of 1?
I do realize that the description says a recommended minibatch size for this exercise is at least 16,
but I was wondering if this was just a recommendation and not mandatory.

Thank you in advance!

**Comment by seominjoon** [comment, marked as answer]
Yes, please feel free to use batch size of your choice, including 1. But I highly recommend at least comparing against it and/or explain why you chose such batch size.

**Reply by iJinjin** [reply]
Okay, thank you!
